server/src/useCases/report/reportSave/handleSaveRequest.ts

```diff
@@ -45,14 +45,18 @@ const RequestSchema = {
   ],
 };
 
-module.exports = class ReportUploadRequestHandler {
-  constructor(saveRequest, response) {
+export default class ReportUploadRequestHandler {
+  public req: any;
+  public res: any;
+  public dbConnect: any;
```

Comment:
Please change req and response to Request and Response types, also replace dbConnect with correct type as in previous request handler

Reply:
done but middleware type is not recognized by typescript (even after the npm types instalation) so I had to extend Request type with "files" and "body"